x-pack/plugins/ml/server/lib/register_settings.ts

```diff
@@ -30,5 +34,41 @@ export function registerKibanaSettings(coreSetup: CoreSetup) {
         }),
       },
     },
+    [ANOMALY_DETECTION_ENABLE_TIME_RANGE]: {
+      name: i18n.translate('xpack.ml.advancedSettings.enableAnomalyDetectionDefaultTimeRangeName', {
+        defaultMessage: 'Enable default time range for anomaly detection jobs.',
```

Comment:
If I'm understanding correctly and these two settings must be used together, I suggest synching the title more closely:
```suggestion
        defaultMessage: 'Enable time filter defaults',
```

Reply:
I changed it to 'Enable time filter defaults for anomaly detection jobs' because this setting is specific to that area 

https://github.com/elastic/kibana/pull/76347/commits/41af491d1fa355395c34966f6c17b1275917b153